src/middlewares/authenticate.js

```diff
@@ -0,0 +1,18 @@
+import jwt from 'jsonwebtoken';
```

Comment:
This is the authorization middleware, basically, we need to use only one because all of us are doing this.

Reply:
I have realized that too @nkpremices